### ISSUE TITLE
[PRE REVIEW]: A general-purpose toolbox for efficient Kronecker-based learning

### ISSUE THREAD

**Issue by whedon** [JuliaCon/proceedings-review#10]
**Submitting author:** @michielstock (<a href="http://orcid.org/0000-0003-0903-6061">Michiel Stock</a>)
**Repository:** <a href="https://github.com/MichielStock/JuliaConSubmission.jl" target ="_blank">https://github.com/MichielStock/JuliaConSubmission.jl</a>
**Editor:** @matbesancon
**Reviewer:** Pending

**Author instructions**

Thanks for submitting your paper to JuliaCon @michielstock. The JuliaCon editor @matbesancon, will work with you on this issue to find a reviewer for your submission before creating the main review issue.

@michielstock if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](http://bit.ly/2V9GSFq) have already agreed to review for JuliaCon and may be suitable for this submission.

**Editor instructions**

The JuliaCon submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @matbesancon it looks like you're currently assigned as the editor for this paper :tada:

For a list of things I can do to help you, just type:

```
@whedon commands
```

**What happens now?**

This submission is currently in a `pre-review` state which means we are waiting for the assigned editor (@matbesancon) to find some reviewers for your submission. This may take anything between a few hours to a couple of weeks. Thanks for your patience :smile_cat:

You can help the editor by looking at [this list of potential reviewers](http://bit.ly/2V9GSFq) to identify individuals who might be able to review your submission (please start at the bottom of the list). Also, feel free to suggest individuals who are not on this list by mentioning their GitHub handles here.

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00010/jcon.00010/10.21105.jcon.00010.pdf)

**Comment by matbesancon** [comment]
Hi @MichielStock, thanks for submitting. It seems the references are not properly linked in the document

**Comment by matbesancon** [comment]
the ref.bib file should be modified accordingly

**Comment by MichielStock** [comment]
Hi @matbesancon, sorry I seem to omit my .bib file. Should now be fine. 

Looking at the list, suitable reviewers could be @Kescobo, @jpfairbanks or @mcognetta (though I think anyone in numerics could be suitable).

**Comment by matbesancon** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List all of Whedon's capabilities
@whedon commands

# Assign a GitHub user as the sole reviewer of this submission
@whedon assign @username as reviewer

# Add a GitHub user to the reviewers of this submission
@whedon add @username as reviewer

# Remove a GitHub user from the reviewers of this submission
@whedon remove @username as reviewer

# List of editor GitHub usernames
@whedon list editors

# List of reviewers together with programming language preferences and domain expertise
@whedon list reviewers

# Change editorial assignment
@whedon assign @username as editor

# Set the software archive DOI at the top of the issue e.g.
@whedon set 10.0000/zenodo.00000 as archive

# Set the software version at the top of the issue e.g.
@whedon set v1.0.1 as version

# Open the review issue
@whedon start review

EDITORIAL TASKS

# Compile the paper
@whedon generate pdf

# Compile the paper from alternative branch
@whedon generate pdf from branch custom-branch-name

# Remind an author or reviewer to return to a review after a
# certain period of time (supported units days and weeks)
@whedon remind @reviewer in 2 weeks

# Ask Whedon to accept the paper and deposit with Crossref
@whedon accept

# Ask Whedon to check the references for missing DOIs
@whedon check references

```

**Comment by matbesancon** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00010/jcon.00010/10.21105.jcon.00010.pdf)

**Comment by MichielStock** [comment]
Proof looks fine.

**Comment by matbesancon** [comment]
@christianpeel would you be available to review?

**Comment by matbesancon** [comment]
@mcognetta would you be available to review this proceeding?

**Comment by christianpeel** [comment]
> @christianpeel would you be available to review?

@matbesancon Yes, I'm happy to review

**Comment by mcognetta** [comment]
@matbesancon Yes, I would be happy to. Are there reviewer instructions and a review template?

**Comment by matbesancon** [comment]
Cool, the review guide is light, located at https://proceedings.juliacon.org/guide/reviewers
Feel free to PR [here](https://github.com/JuliaCon/proceedings.juliacon.org) if you think things are missing. Otherwise we review both the proceeding and the associated software when applying, although more lightly than the Journal of Open-Source software

**Comment by matbesancon** [comment]
@whedon add @mcognetta as reviewer

**Comment by whedon** [comment]
OK, @mcognetta is now a reviewer

**Comment by kescobo** [comment]
Cool! If I'm not needed, I'm going to unfollow this thread. Just tag me if something changes (I'm probably not super appropriate for it though)

**Comment by matbesancon** [comment]
> Cool! If I'm not needed, I'm going to unfollow this thread

yes no problem thanks

**Comment by matbesancon** [comment]
@dmbates would you be available to review this proceeding?

**Comment by matbesancon** [comment]
@jpfairbanks would you be available to review this proceeding?

**Comment by christianpeel** [comment]
@matbesancon   Did you miss my comment above saying that I'm willing to review?   It's not a problem with me if you use other reviewers and skip me.

**Comment by matbesancon** [comment]
@whedon add @christianpeel as reviewer

**Comment by whedon** [comment]
OK, @christianpeel is now a reviewer

**Comment by matbesancon** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/JuliaCon/proceedings-review/issues/13. Feel free to close this issue now!
